docs/immortal-empires/slaanesh.md

```diff
@@ -0,0 +1,33 @@
+# Slaanesh 
+
+## N'Kari - The Arch-Temptor
+
+* **Route I - The Great Game**
+  * Though Slaanesh is presently the least of the four great Ruinous Powers, his brothers secretly fear him, for he feeds on humanity's darkest urges, a source of power that they cannot hope to match. Let the lands of Ulthuan be submerged beneath a writhing, twitching blanket of sin.
+  * Achieve the highest corruption in a minimum of 100 regions
+  * Occupy provinces: 
+    * Caledor, Tiranoc, Nagarythe, Chrace, Cothique, Northern Yvresse, Southern Yvresse, Eataine, Ellyrion, Avelorn and Saphery
+  * **Reward**:
+    * _Slaanesh Ascendant_: Allegiance points: +25% with all factions, Tribute from vassals: +100%, Devotees: +40 per turn
+Alternatives: could also buff things like the Gift of Slaanesh itself, which currently gives +5 devotees and +3 seductive influence per turn PER character's faction, so I stayed away from it because it's harder to control balance-wise
+_Reward flavour text:_ The Dark Prince's power waxes so strongly that it can no longer be contained within the Realms of Chaos. Storms howl outwards from the polar gates, driving his servants onwards to commit countless unspeakable acts in his name.
+
+* **Route II - Paradise Lost**
+  * "Come, mortal. Gorge yourself on the blood of your beloved, slain by your own hands. Yet your lust and despair mingle into a delicious concotion. Then, and only then, will you know the sweet caress of the Dark Prince."
+  * Spread 20 Marks of Slaanesh
+  * Establish 10 Cults
+  * **Reward**:
+    * _Cult Following_: Hero action cost: -30%, Hero action success chance: +20%

```

Comment:
Heroes are the key mechanic of Slaanesh, because you want Marks of Slaanesh. Marks of Slaanesh give you devotees (subfaction resource, very wanted by the player) and accelerates time until you can dominate enemy action (dominate means they instantly become your vassal, even if you were literally at war with them the turn before).

You get marks of slaanesh by doing hero actions on armies. You can't give an army more than one mark of slaanesh. So you basically are doing hero spam. This reward pushes that theme. If too weak, things I would like to add would be hero capacity, hero starter level (both handy AF for getting out more marks of slaanesh). A more difficult reward to implement (maybe impossible?) would be a nice buff to hero movement speed. Hero movement speed = heroes can travel further = spread Marks of Slaanesh across the land more quickly.

Marks of Slaanesh Marks of Slaanesh MARKS OF SLAANESH. That's what the players want MOAR marks = steamroll campaign where you vassalise everyone and everything.